tests/python/gaia-ui-tests/gaiatest/tests/functional/email/manifest.ini

```diff
@@ -4,6 +4,9 @@
 online = true
 smoketest = true
 
+[test_only_one_header_displayed.py]
```

Comment:
The test name is not suggestive enough, but I can't come with a better solution. I will thinks about it and add a comment.